### PR TITLE
add code

### PR DESCRIPTION
**Pull request by DinerMan**

### CODE REVIEW

File: board/gameBoard.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+import (
+	"fmt"
+	"math/rand"
+	"reflect"
+)
+
+
+const color = "\033["
+const colorReset = "0m"
```
Comment (Shanik1) [inline board/gameBoard.go]: If you have several consts, you can create them under one const statement:
```go
const (
 color = "\033["
 colorReset = "0m"
 )
```

Comment (Shanik1) [reply board/gameBoard.go]: Same for vars

File: board/gameBoard.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	for _, row := range g.board {
+		for _, value := range row {
+			fmt.Print(color ,keyMap[value], "m", value, " ")
+		}
+		fmt.Println(color, colorReset)
+	}
+}
+
+func (g GameBoard) CheckWin() bool {
+	needed_color := g.board[0][0]
```
Comment (Shanik1) [inline board/gameBoard.go]: the var convention is camel case (neededColor)

File: main.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"game/board"
+	"strings"
+)
+const size = 20
```
Comment (Shanik1) [inline main.go]: The consts that starts with small letter are internal, and the ones that starts with capital letter are exported. Does the MaxRound needs to be exported?

File: main.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+func validInput(pick string, options []string) bool {
+	for _, val := range options {
+		if pick == val { return true }
+	}
+	return false
+}
+
+func getNumber (options []string) string {
+	var pick string
+	for !validInput(pick, options){
```
Comment (Shanik1) [inline main.go:23]: When you initialized the pick in line 19, it was set to empty string ("")
What is the point of iterate all the options list and see if the pick is equals to one of them? 

Does the point is to see if an option is set to empty string? 
If so, the method should be more explicit to this goal, and the pick option is not relevant

File: board/gameBoard.go [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+		newRow := make([]string, size, size)
+		for y := 0; y < size; y++ {
+			newRow[y] = Keys[rand.Intn(ln)]
+		}
+		newBoard[i] = newRow
+	}
+	return GameBoard{board: newBoard}
+}
+
+func (g GameBoard) PrintBoard() {
```
Comment (Shanik1) [inline board/gameBoard.go]: See the String() method option for go. 
It allows you to determinate how a `fmt.Println(instance)` will look like. 

Replace the PrintBoard method with a String() method representation

File: main.go (hunk 4 above)
Comment (DinerMan) [reply main.go:23]: the method checks if the pick is one of the possible options.
how should I check it?

**Pull request merged**